cypress/support/index.d.ts

```diff
@@ -655,7 +655,12 @@ declare namespace Cypress {
      * @param destinationWindowID - the expected window where the notification redirects
      */
     openNotificationContaining(expectedValue: string | RegExp, destinationWindowID: string | number): Chainable<any>
-
+   /**
+     * Expect the table rows to be greater than a given number
+     *
+     * @param numberOfRows - the number of rows
+     */
+    expectNumberOfRowsToBeGreaterThan(numberOfRows): Chainable<any>
```

Comment:
I'm questioning the need for this function (see my question above)

Reply:
see answer above.